src/shared/Elements/Button.tsx

```diff
@@ -10,17 +10,18 @@ export interface IButtonProps {
 export const Button = styled.button<IButtonProps>`
```

Comment:
Check new UI elements: https://www.figma.com/file/8eJ5e9icjoAOxMRcomRhZV/ui-elements

I think it's nicer to not touch this file, and create a new file for CTA button bc it looks like the only CTA button rn is Logout. Then we can leave this and refactor primary/secondary button styling later